emissionsapi/db.py

```diff
@@ -6,6 +6,7 @@
 """
 
 from functools import wraps
+import logging
```

Comment:
Is "logging" an external package that you found?

Reply:
`logging` is [part of the standard library of Python](https://docs.python.org/3.7/library/logging.html)